Demo/Demo/DemoSettings/DemoSettings.swift

```diff
@@ -20,4 +20,14 @@ enum DemoSettings {
         UserDefaults.standard.string(forKey: DemoTypeDefaultsKey)
             .flatMap { DemoType(rawValue: $0) } ?? .card
     }
+
+    static var clientID: String {
+        switch environment {
+        case .sandbox:
+            return "AdFeO5qt2rodLjSgxSj66sjlBylwpacFtov1VgnptN_VclH1ZNhIcgS1px02mNIOUSEO54B-5drJ7F85"
```

Comment:
How do we want to store these credentials (client ID / secret). I assume we don't actually want them exposed publicly in our demo app? Seems like us having these values just in our app could lead to merchants doing something nefarious (I know it's sandbox, but still probably not a great idea).

Reply:
We _could_ fetch the clientID from the sample server and store the credential on Heroku.

But I honestly think it is fine to leave here - highly doubt a merchant will try to tamper with it or use it.